ui/src/components/App.js

```diff
@@ -21,65 +18,105 @@ import HelpContainer from '../containers/HelpContainer';
 import Main from './Main';
 import LoadingScreen from '../components/LoadingScreen/LoadingScreen';
 
-async function requireAuth() {
-  await store.dispatch(getLoginInfo());
-  const { login } = store.getState();
+import { store } from '../store';
+import { serverIO } from '../serverIO';
 
-  if (login.loggedIn) {
-    await store.dispatch(startSession(login.user.inControl));
```

Comment:
I believe you can now scrap the `startSession` action creator entirely.

Reply:
Oh yes, I wanted to do that, thanks for the reminder :)

Reply:
Seems like it is done.